thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -470,7 +470,9 @@ public Response updateAnomalyFunction(@NotNull @PathParam("id") Long id,
       @QueryParam("filters") String filters,
       @QueryParam("properties") String properties,
       @QueryParam("isActive") Boolean isActive,
-      @QueryParam("frequency") String frequency) throws Exception {
+      @QueryParam("frequency") String frequency,
```

Comment:
can you follow the pattern in 'post' method where you do '1_MINTUES' type of input.